_scripts/main.js

```diff
@@ -19,21 +19,19 @@ function hamburgerColor() {
 
   $hamburgerIcon.addClass('icon-white');
 
-  $(document).scroll(function () {
+  $(document).scroll(function() {
     var scrollPos = window.scrollY;
 
     for (var i = 0; i < $projectSectionArr.length; i++) {
       var $projectSection = $projectSectionArr.eq(i);
-      var offset = $projectSection.offset();
-      var sectionHeight = $projectSection.innerWidth();
-      if (scrollPos > offset && scrollPos < sectionHeight) {
-      //   console.log('hello');
-      //   console.log(offset);
-      //   if ($projectSection.hasClass(.dark)) {
-      //     $hamburgerIcon.addClass('icon-white');
-      //   } else {
-      //     $hamburgerIcon.removeClass('icon-white');
-      //   }
+      var offset = $projectSection.offset().top;
```

Comment:
if you look at the return value of `.offset()`, it returns an object because it gives you both the top offset and the left offset, we just want the top